libtime.h found where LIBTIME_H says (a cross build: in its sysroot),
not only in /usr/include.

--- a/Makefile
+++ b/Makefile
@@ -114,7 +114,10 @@
 
 USE_LIBTIME ?= y#
 
-ifeq ($(wildcard /usr/include/clockd/libtime.h),)
+# where libtime.h is (a cross build: in its sysroot)
+LIBTIME_H ?= /usr/include/clockd/libtime.h
+
+ifeq ($(wildcard $(LIBTIME_H)),)
 CPPFLAGS += -DHAVE_LIBTIME=0
 CPPFLAGS += -DUSE_LIBTIME=0
 else
@@ -183,7 +186,7 @@
 autogenerated_headers:: src/clockd_dbus.inc
 autogenerated_headers:: src/alarmd_config.h
 
-src/clockd_dbus.inc : /usr/include/clockd/libtime.h
+src/clockd_dbus.inc : $(LIBTIME_H)
 	grep > $@ 'define.*CLOCK' $<
 
 src/alarmd_config.h         : src/alarmd_config.h.tpl Makefile
